### PR TITLE
fix issue seven add intro page

### PR DESCRIPTION
**Pull request by SJ682**
fixed issue #7

### CODE REVIEW

**Review by Sakshat682 — COMMENTED** [review]
LGTM

File: intro.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<HTML>
+<HEAD>Intro</HEAD>
+<TITLE>Introduction</TITLE>
+<BODY>
+<H1>Hi we are here o introduce to htmlcss</H1>
+</BODY>
```
Comment (Sakshat682) [inline intro.html:6]: Add some more lines

File: intro.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<HTML>
+<HEAD>Intro</HEAD>
```
Comment (Sakshat682) [inline intro.html]:
```suggestion
<HEAD></HEAD>
```
Remove the header

File: intro.html [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+<TITLE>Introduction</TITLE>
```
Comment (Sakshat682) [inline intro.html:3]: Change the title

File: intro.html (hunk 2 above)
Comment (SJ682) [reply intro.html]: Ok i will do that

**Comment by SJ682** [comment]
Please review it

**Review by Sakshat682 — APPROVED** [review]
Yes change that

**Review by Sakshat682 — APPROVED** [review]

**Comment by SJ682** [comment]
Thnx for approving and merging my PR @Sakshat682

**Pull request merged**